Pyrado/scripts/training/qq-su_npdr_gaussian-prior.py

```diff
@@ -111,7 +112,7 @@
     }
 
     # Transform the domain parameter space
-    env_sim = SqrtDomainParamTransform(env_sim, [dp_name for dp_name in dp_mapping.values()])
+    env_sim = DomainParamTransform(env_sim, [dp_name for dp_name in dp_mapping.values()], SqrtTransformation())
```

Comment:
Please check if this script runs

Reply:
It does not, but neither does it on master.

```
Traceback (most recent call last):
  File "SimuRLacra/Pyrado/scripts/training/qq-su_npdr_gaussian-prior.py", line 204, in <module>
    **algo_hparam,
  File "SimuRLacra/Pyrado/pyrado/algorithms/meta/npdr.py", line 69, in __init__
    super().__init__(*args, num_checkpoints=3, init_checkpoint=-1, **kwargs)
  File "SimuRLacra/Pyrado/pyrado/algorithms/meta/sbi_base.py", line 240, in __init__
    self._setup_sbi(prior=prior)
  File "SimuRLacra/Pyrado/pyrado/algorithms/meta/sbi_base.py", line 303, in _setup_sbi
    self._sbi_simulator, self._sbi_prior = prepare_for_sbi(rollout_sampler, prior)
  File ".anaconda3/envs/pyrado/lib/python3.7/site-packages/sbi/utils/user_input_checks.py", line 501, in prepare_for_sbi
    check_sbi_inputs(simulator, prior)
  File ".anaconda3/envs/pyrado/lib/python3.7/site-packages/sbi/utils/user_input_checks.py", line 518, in check_sbi_inputs
    simulation = simulator(theta)
  File ".anaconda3/envs/pyrado/lib/python3.7/site-packages/sbi/utils/user_input_checks.py", line 434, in batch_loop_simulator
    xs = list(map(simulator, theta))
  File ".anaconda3/envs/pyrado/lib/python3.7/site-packages/sbi/utils/user_input_checks.py", line 399, in pytorch_simulator
    return torch.as_tensor(simulator(theta), dtype=float32)
  File "SimuRLacra/Pyrado/pyrado/sampling/sbi_rollout_sampler.py", line 317, in __call__
    check_domain_params(ro_sim, dp_value, self.dp_names)
  File "SimuRLacra/Pyrado/pyrado/sampling/sbi_rollout_sampler.py", line 545, in check_domain_params
    msg="The domain parameters after the rollouts are not identical to the ones commanded by the sbi!"
pyrado.utils.exceptions.ValueErr: The domain parameters after the rollouts are not identical to the ones commanded by the sbi!

Process finished with exit code 1

```